.travis.yml

```diff
@@ -103,8 +107,6 @@ script:
         done;
       fi
     - cd .. && mv julia julia2
-    - cp /tmp/julia/lib/julia/sys.ji local.ji && /tmp/julia/bin/julia -J local.ji -e 'true' &&
-        /tmp/julia/bin/julia-debug -J local.ji -e 'true' && rm local.ji
```

Comment:
we don't actually want to be installing sys.ji (which is a build intermediate). otoh, this command is now being run as part of bootstrapping, so it should be redundant to run it again as part of the tests

this test, added by e92fc781f3b529bcc3d3c0502c33d361a8cb3f5f, was obsoleted by f0712f0dfd96adf468cdfc1268bbf7055d79a36f which removed the code path that this was supposed to test (https://github.com/JuliaLang/julia/commit/47767573d2f9891cdc5db43fc2dcf98cb2503535#commitcomment-9240188)